src/typeDefs/inputs.ts

```diff
@@ -459,4 +459,13 @@ export const inputs = gql`
     imageUrl: String
     videoUrl: String
   }
+
+  input CreateAdvertisementInput {
+    endDate: Date!
+    name: String!
+    organizationId: ID!
+    startDate: Date!
+    type: String!
```

Comment:
```graphql
enum AdvertisementType {
  POP_UP
  MENU
  BANNER
}
```
declare this and use this as the advertisement's `type` field everywhere in the schema